packages/vue/src/components/organisms/SfSidebar/SfSidebar.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <div class="sf-sidebar" :class="[staticClass, className]">
+  <div class="sf-sidebar">
```

Comment:
What do you think about adding here: 
```
:class="{'sf-sidebar--right': position === 'right'}"
```
to change class on changing the position value?

Reply:
make sense, when `sf-sidebar--righ` class will not be added to `SfSidebar`